### PR TITLE
chore(demo): correct gap between push in basic example

### LINKED ISSUE #9060: 🐞 - broken gap
### Playground Link

/components/push

### Description

image

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by shiv9604**
Close #9060

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/push/examples/1/index.less [hunk 1]
```diff
@@ -10,6 +10,14 @@
     max-inline-size: 100%;
     border-radius: var(--tui-radius-l);
     margin-bottom: 1rem;
+
```
Comment (waterplea) [inline projects/demo/src/modules/components/push/examples/1/index.less:13]: Please use the fix I suggested in the issue. The problem is not with gap per say, it's the background of the wrapper that's missing. I believe the gap is correct.

Comment (shiv9604) [reply projects/demo/src/modules/components/push/examples/1/index.less:13]: A wrapper contains 2 push elements as child elements. If we use the suggested fix, it merges both push elements. Can you confirm is wrapper necessary for each push element?

Comment (waterplea) [reply projects/demo/src/modules/components/push/examples/1/index.less:13]: Yes, the idea of this example is to show a list of push elements as in a dropdown when clicking on Bell icon with notifications, hence first one having (X) icon as if it's there to close the whole list.

Comment (shiv9604) [reply projects/demo/src/modules/components/push/examples/1/index.less]: Thanks for clarifying the requirements Alex, I have resolved the PR comment. Please checkout.

**Review by waterplea — APPROVED** [review]

**Pull request merged**